vars/utils.groovy

```diff
@@ -106,18 +106,18 @@ def finApplicability(){
         env.gateway_user = "${gateway_user}"
         env.PYTHONPATH=env.WORKSPACE
         env.PATH="${env.WORKSPACE}/tuna:${env.PATH}"
-        env.OTEL_METRICS_EXPORTER="none"
-        env.OTEL_TRACES_EXPORTER="console"
+        //env.OTEL_METRICS_EXPORTER="none"
+        //env.OTEL_TRACES_EXPORTER="console"
```

Comment:
@cderb  and I discussed this. He was getting some new errors and I didnt want to put the debugging on him so I told him to comment it out for now. We can probably remove it altogether since we dont have the backend set up yet to actually look at the traces.